### PR TITLE
Update rubygem-fog-aws to 3.3.0

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like we'll need to update fog-json and fog-core for this.

**Comment by ekohl** [comment]
That means we'll need https://github.com/theforeman/foreman/pull/6475 so core itself actually knows how to deal with it.

**Comment by ares** [comment]
these version are compatible with fog-google 1.8.2 and fog_azure_rm discussed in the core PR, fog-google and its dependencies is yet to be packaged

File: packages/foreman/rubygem-net-ssh/rubygem-net-ssh.spec [hunk 1]
```diff
@@ -6,14 +6,13 @@
...
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.1
-Release: 6%{?dist}
+Version: 4.2.0
+Release: 1%{?dist}
 Group: Development/Languages
 License: MIT
 URL: https://github.com/net-ssh/net-ssh
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 # Fixes #20018
```
Comment (ekohl) [inline packages/foreman/rubygem-net-ssh/rubygem-net-ssh.spec]: Can you drop this line?

**Comment by ares** [comment]
Closed in favor of https://github.com/theforeman/foreman-packaging/pull/3545

**Pull request closed without merging**